CHANGELOG.md

```diff
@@ -46,3 +46,7 @@ Version 2.0.0
 Version 2.1.0
 
                 Added support for python 3.11 support. Removed django 3.2 support.
+
+Version 2.2.0
```

Comment:
We should update major version as we are dropping support of python version

Reply:
ok got it